LICENSE-binary

```diff
@@ -504,8 +504,7 @@ javax.cache:cache-api:1.1.1
 javax.servlet:javax.servlet-api:3.1.0
 javax.servlet.jsp:jsp-api:2.1
 javax.websocket:javax.websocket-api:1.0
-javax.ws.rs:jsr311-api:1.1.1
```

Comment:
why remove it?

Reply:
dependency was removed during the move to jersey v2 - we now have rs-api dependency instead

Reply:
Alright, JavaEE/JakartaEE API dependencies are a mess...